tutorial/10_key_features/005_visualization.py

```diff
@@ -77,7 +80,7 @@ def objective(trial):
     }
 
     # Add a callback for pruning.
-    pruning_callback = optuna.integration.LightGBMPruningCallback(trial, "auc")
+    pruning_callback = optuna_integration.LightGBMPruningCallback(trial, "auc")
```

Comment:
As mentioned above, we would like to avoid installing optuna-integration in Optuna's CI. How about simply stopping the use of PruningCallback? The execution time will increase because we won't be pruning, but can we accept this?